velox/dwio/dwrf/test/CacheInputTest.cpp

```diff
@@ -62,14 +62,14 @@ class CacheTest : public testing::Test {
   }
 
   void TearDown() override {
```

Comment:
Can you add SSDCache test to verify the ssd cache can't write after shutdown? We shall throw in that case? Thanks!